apps/spruce/src/pages/image/getTabTitle.ts

```diff
@@ -0,0 +1,7 @@
+import { ImageTabRoutes } from "constants/routes";
+
+export const getTabTitle = (tab: ImageTabRoutes): { title: string } =>
+  ({
```

Comment:
nit: This would read better as as switch statement. The switch statement also has a built in handler for the default case.